docs/shared/migration/angular-integrated.md

```diff
@@ -0,0 +1,103 @@
+>
+> Currently, the automated migration supports workspaces using the following executors (builders):
+>
+> - `@angular-devkit/build-angular:browser`
+> - `@angular-devkit/build-angular:dev-server`
+> - `@angular-devkit/build-angular:extract-i18n`
+> - `@angular-devkit/build-angular:karma`
+> - `@angular-devkit/build-angular:ng-packagr`
+> - `@angular-devkit/build-angular:protractor`
+> - `@angular-devkit/build-angular:server`
+> - `@angular-eslint/builder:lint`
+> - `@cypress/schematic:cypress`
+> - `@nguniversal/builders:prerender`
+> - `@nguniversal/builders:ssr-dev-server`
+>
+> Support for other executors may be added in the future.
+
+This installs the `@nrwl/angular` (or `@nrwl/workspace`) package into your workspace and runs a generator (or schematic) to transform your workspace. The generator applies the following changes to your workspace:
+
+- Installs the `nx` and `@nrwl/workspace` packages.
+- Creates an `nx.json` file in the root of your workspace.
+- Creates configuration files for Prettier.
+- Creates an `apps` folder for generating applications.
+- Creates a `libs` folder for generating libraries.
+- Creates a `tools` folder that includes files for custom workspace tooling, such as workspace-specific generators and scripts.
+- Moves your applications into the `apps` folder, and updates the relevant file paths in your configuration files.
+- Moves your e2e suites into the `apps/<app name>-e2e` folder, and updates the relevant file paths in your configuration files.
+- Moves your libraries into the `libs` folder, and updates the relevant file paths in your configuration files.
+- Updates your `package.json` scripts to use `nx` instead of `ng`.
+- Splits your `angular.json` into `project.json` files for each project with updated paths.
+
+After the changes are applied, your workspace file structure should look similar to the one below:
+
+```text
+<workspace name>/
+├── apps/
+│   └─ <app name>/
+│       ├── src/
+│       │   ├── app/
+│       │   ├── assets/
+│       │   ├── favicon.ico
+│       │   ├── index.html
+│       │   ├── main.ts
+│       │   └── styles.css
+│       ├── project.json
+│       ├── tsconfig.app.json
+│       └── tsconfig.spec.json
+├── libs/
+│   └── <lib name>/
+│       ├── src/
+│       ├── ng-package.json
+│       ├── package.json
+│       ├── project.json
+│       ├── README.md
+│       ├── tsconfig.lib.json
+│       ├── tsconfig.lib.prod.json
+│       └── tsconfig.spec.json
+├── tools/
+├── .editorconfig
+├── .gitignore
+├── .prettierignore
+├── .prettierrc
+├── karma.conf.js
+├── nx.json
+├── package.json
+├── README.md
+└── tsconfig.base.json
+```
+
+Your workspace is now powered by Nx! You can verify that your application still runs as intended:
+
+- To serve, run `ng serve <app name>` (or `nx serve <app name>`).
```

Comment:
Should we just recommend using `nx ...`? I'm not even sure `ng` would work in all situations.

I'd phrase this more like a "before" and "after". Where they'd replace just `ng` with `nx` and that's it.

Reply:
I also think we shouldn't promote continue using `ng` and we should promote replacing it with `nx`. That way we make it more apparent it's a different CLI.
We no longer generate the script to decorate the Angular CLI.